app/models/page_content_register.rb

```diff
@@ -17,7 +19,10 @@ def new_blocks_attributes
                         else
                           @new_blocks_attributes
                         end
-    return @new_blocks_attributes if home_page? || footer?
+
+    if home_page? || footer?
+      return blocks_attributes + srcset_blocks(blk_attrs: blocks_attributes)
```

Comment:
_srcset_blocks_ may not be the best so I will switch it to __new_srcset_blocks_.